community/modules/scripts/htcondor-install/files/autoscaler.py

```diff
@@ -194,11 +194,26 @@ def scale(self):
             REQUEST_MEMORY_ATTRIBUTE,
         ]
 
+        instanceTemplateInfo = self.getInstanceTemplateInfo()
+        self.is_spot = instanceTemplateInfo["is_spot"]
+        self.cores_per_node = instanceTemplateInfo["guest_cpus"]
+        print(f"MIG is configured for Spot pricing: {self.is_spot}")
+        print("Number of CPU per compute node: " + str(self.cores_per_node))
```

Comment:
nit. Let's use f-string to conform with previous line and avoid `str()`

Reply:
ack. created a task in backlog that addresses this alongside future development plans.